terraform/modules/helm/velero/variables.tf

```diff
@@ -121,4 +122,5 @@ variable "velero_sa_annotations" {
 variable "gcp_project_id" {
   type        = string
   description = "Google Project ID, if the provider is gcp"
-}
\ No newline at end of file
+  default     = ""
```

Comment:
Why do we need an emptry String as default?

Reply:
needed regarding gcp changes, same as velero_sa_iam_role_name